glommio/src/free_list.rs

```diff
@@ -47,7 +47,7 @@ impl<T> Default for FreeList<T> {
     fn default() -> Self {
         FreeList {
             first_free: None,
-            slots: Vec::new(),
+            slots: Vec::with_capacity(8 << 10),
```

Comment:
Why not `1 << 13`?

Reply:
I personally find `8 << 10` way more readable. I am a bit dumb so I have to do math in my head to remember how much 13 shift is. It's not *that* hard, because from my Linux days I had "12" memorized (page size), but still.
10 is a kB, so "8 << 10" immediately evokes 8kB. Of course this is all subjective, because I had to memorize that 10 is a kB as well, and I can say all day this is "more natural", but at the end of the day that obviously means nothing.

Reply:
Ahaha I'm the same as @glommer and this was going to be my response. I'm fine changing this to `1 << 13` if you feel strongly about it. Eventually, I'll remember what value it ends up being :laughing:

Reply:
I don't feel strongly about it at all, I was just asking out of curiosity :)